src/routes/Acknowledgements.svelte

```diff
@@ -27,8 +27,8 @@
 
 <HeaderWithAnchor id="project-team" level=2>Project Team</HeaderWithAnchor>
 <ul>
-  <li><strong>GSA</strong>: Syed Azeem, Karl Hebenstreit, Joseph Novak, Charles Popelka, Richard Speidel, Sean Zerges.</li>
-  <li><strong>CivicActions</strong>: Owen Barton, Danita Delce, Mike Gifford, Marlena Medford, Daniel Mundra, Farooq Zakhilwal.</li>
+  <li><strong>GSA</strong>: Syed Azeem, Karl Hebenstreit, Andrew Neilson, Joseph Novak, Charles Popelka, Richard Speidel, Sean Zerges.</li>
+  <li><strong>CivicActions</strong>: Owen Barton, Melinda Burgess, Danita Delce, Mike Gifford, Marlena Medford, Daniel Mundra, Jacqueline Quintanilla, Farooq Zakhilwal.</li>
```

Comment:
Just realized this is alphabetical by last name. Probably should eventually re-org the CONTRIBUTION.txt file & AUTHORS.txt to align with this. I ordered it by first name.